src/Adapter/Order/Repository/OrderRepository.php

```diff
@@ -71,4 +77,37 @@ public function get(OrderId $orderId): Order
 
         return $order;
     }
+
+    /**
+     * @param OrderDetailId $detailId
+     * @param LanguageId $languageId
+     *
+     * @return OrderDetailCustomizations|null
+     *
+     * @throws PrestaShopException
+     * @throws \PrestaShopDatabaseException
+     */
+    public function getOrderDetailCustomizations(OrderDetailId $detailId, LanguageId $languageId): ?OrderDetailCustomizations
+    {
+        $orderDetail = new OrderDetail($detailId->getValue());
+        $order = new Order($orderDetail->id_order);
+        $customizations = [];
```

Comment:
just a nitpick if you have other things to fix (not blocking)
This var will end up unused if (!is_array($customizedDatas)).
So ideally it should be moved before foreach loop to avoid initializing it for no wreason